Makefile: Fix static linking

Since libsndfile uses funtions from libm, the -lm should be specified
after libsndfile for static linking.

Signed-off-by: "Eric Limpens" <Limpens@gmail.com>

diff -purN pifmrds-0bf57f9ce0d954365a38d8af8e7be6f28521c3f2.orig/src/Makefile pifmrds-0bf57f9ce0d954365a38d8af8e7be6f28521c3f2/src/Makefile
--- pifmrds-0bf57f9ce0d954365a38d8af8e7be6f28521c3f2.orig/src/Makefile	2016-02-26 08:06:43.102962592 +0100
+++ pifmrds-0bf57f9ce0d954365a38d8af8e7be6f28521c3f2/src/Makefile	2016-02-26 08:27:11.069148203 +0100
@@ -1,10 +1,10 @@
 CC = gcc
 
 app: rds.o waveforms.o pi_fm_rds.o fm_mpx.o control_pipe.o
-	$(CC) $(LDFLAGS) -o pi_fm_rds rds.o waveforms.o pi_fm_rds.o fm_mpx.o control_pipe.o -lm -lsndfile
+	$(CC) $(LDFLAGS) -o pi_fm_rds rds.o waveforms.o pi_fm_rds.o fm_mpx.o control_pipe.o -lsndfile -lm
 
 rds_wav: rds.o waveforms.o rds_wav.o fm_mpx.o
-	$(CC) $(LDFLAGS) -o rds_wav rds_wav.o rds.o waveforms.o fm_mpx.o -lm -lsndfile
+	$(CC) $(LDFLAGS) -o rds_wav rds_wav.o rds.o waveforms.o fm_mpx.o -lsndfile -lm
 
 rds.o: rds.c waveforms.h
 	$(CC) $(CFLAGS) rds.c
